schemas/Clinical/index.js

```diff
@@ -146,39 +136,26 @@ const typeDefs = gql`
     """
     commitClinicalRegistration(shortName: String!, registrationId: String!): [String]!
       @cost(complexity: 20)
-    
+
     """
     Upload Clinical Submission files
     """
     uploadClinicalSubmissions(
-      shortName: String!,
+      shortName: String!
       clinicalFiles: [Upload!]
     ): ClinicalSubmissionData! @cost(complexity: 30)
   }
 `;
 
-const convertRegistrationRecordToGql = (record, row) => ({
-  row,
-  programShortName: record.program_id,
-  donorSubmitterId: record.submitter_donor_id,
-  specimenSubmitterId: record.submitter_specimen_id,
-  specimentType: record.specimen_type,
-  sampleSubmitterId: record.submitter_sample_id,
-  sampleType: record.sample_type,
-  gender: record.gender,
-  tumourNormalDesignation: record.tumour_normal_designation,
-  data: () =>
-    omit(record, [
-      'program_id',
-      'submitter_donor_id',
-      'submitter_specimen_id',
-      'specimen_type',
-      'submitter_sample_id',
-      'sample_type',
-      'tumour_normal_designation',
-      'gender',
-    ]),
-});
+const convertRegistrationRecordToGql = (record, row) => {
```

Comment:
So now both registration and submission follow this setup?

Reply:
yes

Reply:
need the exact heading names as provided by clinical service to get passed through to ui. should've done it this way in first place